tests/Doctrine/Tests/ORM/Functional/Ticket/GH10049/SetInheritedReadOnlyPropertyValueTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket\GH10049;
+
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+/**
+ * @requires PHP 8.1
+ */
+class SetInheritedReadOnlyPropertyValueTest extends OrmFunctionalTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->createSchemaForModels(
+            ReadOnlyPropertyOwner::class,
+            ReadOnlyPropertyInheritor::class
+        );
+    }
+
+    public function test(): void
```

Comment:
> not sure about the naming

I find it always better when the name of the test describes what you are testing. Let me know if I understood correctly.

```suggestion
    public function testItDoesNotCrashOnInheritedReadonlyProperty(): void
```

Reply:
Yes but here I'm not testing a class but a specific behavior so the class name is what is tested 🤔

Reply:
Oh right sorry, didn't notice!

Reply:
That behaviour could be part of the name. I wouldn't be able to understand the test without the context that this Pr is giving.

Reply:
Is the class name not enough?

Do you want me to switch with the method name (and rename the class something like `GH10049Test`)?

Reply:
That would also be a good idea :+1: